rfcs/text/0033/process.yml

```diff
@@ -51,6 +51,44 @@
 
         Best efforts are made to ensure complete lines are captured in these events. Assumptions should NOT be made that multiple lines will appear in the same event. TTY output may contain terminal control codes such as for cursor movement, so some string queries may not match due to terminal codes inserted between characters of a word.
 
+    - name: io.total_bytes_captured
+      level: extended
+      type: number
+      description: >
+        The total number of bytes captured in this event.
+
+    - name: io.total_bytes_skipped
+      level: extended
+      type: number
+      description: >
+        The total number of bytes skipped in this event.
```

Comment:
Perhaps add clarity to help implementors:

"The total number of bytes of output that were not captured due to implementation restrictions such as buffer size limits.  Implementors should strive to ensure this value is always zero"